### PR TITLE
feat: dockerize

### PR DESCRIPTION
**Pull request by bstaeheli**
This PR is to dockerize this solution

Since a few months it runs stable at our environment.

### CODE REVIEW

**Comment by martinholters** [comment]
Ref. https://hub.docker.com/r/martinholters/rscp2mqtt (https://github.com/martinholters/rscp2mqtt-docker) for an alternative.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+COPY ./ /tmp/rscp2mqtt
+WORKDIR /tmp/rscp2mqtt
+RUN make
+
+RUN mkdir -p /opt/rscp2mqtt
+RUN cp -a rscp2mqtt /opt/rscp2mqtt
+RUN cp config.template /opt/rscp2mqtt/.config
+RUN chown -R nobody:99 /opt/rscp2mqtt
+
+# Switch to use a non-root user from here on
```
Comment (martinholters) [inline Dockerfile:20]: How about a staged build to have a smaller image in the end, like so:

```suggestion
FROM alpine
RUN apk --no-cache add libstdc++ mosquitto-libs
COPY --from=0 /opt/rscp2mqtt /opt/rscp2mqtt

# Switch to use a non-root user from here on
```

**Comment by bstaeheli** [comment]
> Ref. https://hub.docker.com/r/martinholters/rscp2mqtt (https://github.com/martinholters/rscp2mqtt-docker) for an alternative.

Hi @martinholters,
I already saw your project and used it as an inspiration. I prefer to have the docker file directly in the project. It makes the development of the project much easier.

**Comment by martinholters** [comment]
@pvtom What do you think about adding a Github Actions workflow here that builds (and uploads) an image for every push to main and every tag? If you're interested, I can try to implement it and open a PR.

**Comment by pvtom** [comment]
Hi Martin, good idea.  👍

**Comment by martinholters** [comment]
See #18

**Pull request merged**